### ISSUE TITLE
Not a functionality issue

### ISSUE THREAD

**Issue by abhinav272** [droidster1989/DroidsterUtilityDemo#1]
Nice, It is always great to have utility functions to accelerate app development so kudos to your effort. :+1:
But I would like to suggest you a small change, you must segregate the Utility file into sperate files based on their functionality as Bob Martin stated in his SOLID Principles that a file should have a single responsibility.
So you can have a separate file for UI specific util functions, similarly separate file for formatting of string, dates, and a separate file for Network related functions etc.

I hope you got what I'm trying to say.
This would make your growing codebase more readable, maintainable and testable.

**Comment by droidstered** [comment]
Yes sure. Thanks for your feedback. I have just started to use Github.

Will back to with some changes as u suggested.
